experimental/pagerduty-webhook/program.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import requests
+import json
+import click
+
+
+@click.command()
+@click.option('--authorizationtoken', prompt='API token', help='PagerDuty API token')
+def cli(authorizationtoken):
+    """Simple program that adds a webhook to Pager Duty ."""
+    headers = getHeaders(authorizationtoken)
+    url = "https://api.pagerduty.com"
+    for key, value in get_services(headers, url).items():
+        create_webhook_subscription(key, value, url, headers)
+
+
+def create_webhook_subscription(id, name, url, headers):
```

Comment:
Should we add a check for existing webhook subscriptions? If re-run it looks like this would create duplicate webhook subscriptions for every service? Unless Pager Duty is smart enough to... not do that.

Reply:
Found the way bit more looping but will do for us...